src/snowflake/cli/plugins/nativeapp/exceptions.py

```diff
@@ -63,11 +63,11 @@ def __init__(self, identifier: str):
 
 
 class CouldNotFindDistributionForAppPackage(ClickException):
-    """The 'distribution' attribute for an app package could not be found."""
+    """The 'distribution' attribute for an application package could not be found."""
```

Comment:
Maybe: "The 'distribution' attribute has not been set for an application package."

"Not found" makes me think that there was some other error (e.g. an internal error.). The action to resolve this would be to set the distribution property, no?

Reply:
This exception is actually not being used anywhere in the code. I'll remove it from here.